pkg/cmd/server/start/master_args.go

```diff
@@ -84,6 +86,8 @@ func BindMasterArgs(args *MasterArgs, flags *pflag.FlagSet, prefix string) {
 
 	flags.StringVar(&args.EtcdDir, prefix+"etcd-dir", "openshift.local.etcd", "The etcd data directory.")
 
+	flags.StringSliceVar(&args.APIServerCAFiles, prefix+"certificate-authority", args.APIServerCAFiles, "Optional files containing signing authorities to use (in addition to the generated signer) to verify the API server's serving certificate.")
```

Comment:
> not sure how I feel about this showing up here...

I think I'm ok with it.   The level of pain involved in regenerating certs is high enough to warrant having either this or a separate `write-config` command.